tests/behaviour/expectations/behaviour.ts

```diff
@@ -222,67 +222,21 @@ export const expectations = flatten(
             Expect.Simple('explicit', ['20'], ['20', '20', '0']),
             Expect.Simple('explicit', ['240'], ['240', '240', '0']),
           ]),
-        ]),
-        new Dir('cross_contract_calls', [
```

Comment:
A lot of important tests are getting deleted here. Could you please reverse these deletions